pkg/scheduler/apis/config/validation/validation_pluginargs.go

```diff
@@ -29,7 +29,7 @@ import (
 )
 
 // ValidateDefaultPreemptionArgs validates that DefaultPreemptionArgs are correct.
-func ValidateDefaultPreemptionArgs(args config.DefaultPreemptionArgs) error {
+func ValidateDefaultPreemptionArgs(args *config.DefaultPreemptionArgs) error {
```

Comment:
we should be passing a path to all these functions

Reply:
Done